src/main/java/com/google/api/codegen/config/FlatteningConfig.java

```diff
@@ -329,4 +329,13 @@ private static String flatteningConfigToString(FlatteningConfig flatteningConfig
     paramList.forEach(p -> paramsAsString.append(p.getSimpleName()).append(", "));
     return paramsAsString.toString();
   }
+
+  /** Return if the flattening config contains a parameter that is a resource name. */
```

Comment:
This was copied from JavaMethodViewGenerator; only the `public static` method modifiers were added.